radio/src/telemetry/crossfire.h

```diff
@@ -41,4 +41,21 @@ void processCrossfireTelemetryData(uint8_t data);
 void crossfireSetDefault(int index, uint8_t id, uint8_t subId);
 bool isCrossfireOutputBufferAvailable();
 
+#if defined(PCBX7)
```

Comment:
Is there a good reason to restrict this to X7? I vaguely that some user with a Horus also had problem with Crossfire.